### ISSUE TITLE
security considerations ?

### ISSUE THREAD

**Issue by fschaer** [HEP-Puppet/htcondor#42]
Hi,

the template 10_security.config.erb contains this line:

ALLOW_ADMINISTRATOR = *@$(UID_DOMAIN)/$(FULL_HOSTNAME)

I tried as a lambda user : I was allowed to condor,_restart and I'm assuming that I would be allowed to switch it down if I wanted... this makes me believe that any job would be allowed to do that too.

I am right or wrong ?
If right : how can we fix this hole ?

Thanks

**Comment by kreczko** [comment]
Yes, thank you for the reminder. I have a fixed version on a feature branch, but this is something that should be fixed quicker.
The fix is to change
```
ALLOW_ADMINISTRATOR = *@$(UID_DOMAIN)/$(FULL_HOSTNAME)
```
to
```
ALLOW_ADMINISTRATOR = root@$(UID_DOMAIN)/$(IP_ADDRESS), condor_pool@$(UID_DOMAIN)/$(IP_ADDRESS), $(CMS)
```
where CMS is the collection of central managers allowing for condor & condor_pool users.

**Comment by fschaer** [comment]
thanks, I will create a local branch for this, and delete when fixed upstream

**Comment by kreczko** [comment]
Fixed in #52
